filemanager/__init__.py

```diff
@@ -16,290 +13,437 @@
+    action = forms.ChoiceField(choices=ActionChoices)
+    path = forms.CharField(max_length=200, required=False)
+    name = forms.CharField(max_length=32, required=False)
+    current_path = forms.CharField(max_length=200, required=False)
+    file_or_dir = forms.CharField(max_length=4)
+
 
 class FileManager(object):
-  """
-  maxspace,maxfilesize in KB
-  """
-  idee = 0
-  def __init__(self,basepath,ckeditor_baseurl='',maxfolders=50,maxspace=5*1024,maxfilesize=1*1024,public_url_base=None,extensions=None):
-    if basepath[-1] == '/':
-      basepath = basepath[:-1]
-    if ckeditor_baseurl and ckeditor_baseurl[-1] == '/':
-      ckeditor_baseurl = ckeditor_baseurl[:-1]
-    self.basepath = basepath
-    self.ckeditor_baseurl = ckeditor_baseurl
-    self.maxfolders = maxfolders
-    self.maxspace = maxspace
-    self.maxfilesize = maxfilesize
-    self.extensions = extensions;
-    self.public_url_base = public_url_base
+    """
+    maxspace,maxfilesize in KB
+    """
+    idee = 0
 
-  def rename_if_exists(self,folder,file):
-    if folder[-1] != os.sep:
-      folder = folder + os.sep
-    if os.path.exists(folder+file):
-      if file.find('.') == -1:
-        # no extension
-        for i in range(1000):
-          if not os.path.exists(folder+file+'.'+str(i)):
-            break
-        return file+'.'+str(i)
-      else:
-        extension = file[file.rfind('.'):]
-        name = file[:file.rfind('.')]
-        for i in range(1000):
-          if not os.path.exists(folder+name+'.'+str(i)+extension):
-            break
-        return name + '.' + str(i) + extension
-    else:
-      return file
+    def __init__(
+        self,
+        basepath,
+        ckeditor_baseurl='',
+        maxfolders=50,
+        maxspace=5*1024,
+        maxfilesize=1*1024,
+        public_url_base=None,
+        extensions=None,
+    ):
+        if basepath[-1] == '/':
+            basepath = basepath[:-1]
+        if ckeditor_baseurl and ckeditor_baseurl[-1] == '/':
+            ckeditor_baseurl = ckeditor_baseurl[:-1]
+        self.basepath = basepath
+        self.ckeditor_baseurl = ckeditor_baseurl
+        self.maxfolders = maxfolders
+        self.maxspace = maxspace
+        self.maxfilesize = maxfilesize
+        self.extensions = extensions
+        self.public_url_base = public_url_base
 
-  def get_size(self,start_path):
-    total_size = 0
-    for dirpath, dirnames, filenames in os.walk(start_path):
-      for f in filenames:
-        fp = os.path.join(dirpath, f)
-        total_size += os.path.getsize(fp)
-    return total_size
+    def rename_if_exists(self, folder, file):
+        if folder[-1] != os.sep:
+            folder = folder + os.sep
+        if os.path.exists(folder+file):
+            if file.find('.') == -1:
+                # no extension
+                for i in range(1000):
+                    if not os.path.exists(folder+file+'.'+str(i)):
+                        break
+                return file+'.'+str(i)
```

Comment:
@ErwinJunge Can you add spaces before and after a '+'. Here and everywhere?

Reply:
Done. I prefer spaces around operators as well, but saw it so much in the file that I kind of assumed it was the accepted codestyle for the project. And since flake8 didn't complain, I left it in.